tests/ArrayAssignmentTest.php

```diff
@@ -797,7 +797,7 @@ public function offsetSet($offset, $value): void
                     $a = null;
                     $a[0][] = 1;',
                 'assertions' => [
-                    '$a' => 'array{0: non-empty-list<int>}',
+                    '$a' => 'array{non-empty-list<int>}',
```

Comment:
This is more correct — using explicit `int` keys implies the keys don't have to be ordered